### ISSUE TITLE
UnsupportedCommandException: mouseMoveTo on Selenium 3.6.0 IE browser

### ISSUE THREAD

**Issue by Nikunj1703** [SeleniumHQ/selenium#5009]
## Meta -
OS:  Windows 7 Enterprise (64 bit)
Selenium Version:  
3.6.0
Browser:  
Internet Explorer

Browser Version:  
IE10 (IEDriverServer 3.6.0) using 32 bit

## Expected Behavior -
mouseMoveTo of Actions API should be supported on IE10
## Actual Behavior -
Fails to perform mouseMoveTo. Following is the log:
```
Caused by: org.openqa.selenium.UnsupportedCommandException: mouseMoveTo
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.http.AbstractHttpCommandCodec.encode(AbstractHttpCommandCodec.java:222)
	at org.openqa.selenium.remote.http.AbstractHttpCommandCodec.encode(AbstractHttpCommandCodec.java:119)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:646)
	at org.openqa.selenium.remote.RemoteExecuteMethod.execute(RemoteExecuteMethod.java:35)
	at org.openqa.selenium.remote.RemoteMouse.mouseMove(RemoteMouse.java:89)
	at org.openqa.selenium.interactions.MoveMouseAction.perform(MoveMouseAction.java:43)
	at org.openqa.selenium.interactions.CompositeAction.perform(CompositeAction.java:36)
	at org.openqa.selenium.interactions.Actions$BuiltAction.perform(Actions.java:641)
	at org.openqa.selenium.interactions.Actions.perform(Actions.java:594)
```

## Steps to reproduce -
1. In the below code, finding a webElement
2. Using the actions API and perform move to the element
```
try{
      WebElement el = driver.findElement(by);
      Actions builder = new Actions(driver);
      builder.moveToElement(el).perform();
}catch(Exception ex){
      logger.error("Exception: {}", ex);
}
```
3. Observe the exception:
```
Exception: org.openqa.selenium.UnsupportedCommandException: mouseMoveTo
```

**Comment by barancev** [comment]
Please provide a [concise reproducible test case](http://sscce.org/) and describe what results you are seeing (including complete log) and what results you expect.

See [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#issue-contributions)

**Comment by Nikunj1703** [comment]
@barancev I did not run into the issue using 3.7.1. I think we can close this ticket. Thanks for your help!

**Comment by timrsfo** [comment]
this was happening to me. for me it was a maven pom.xml issue. I had several selenium jars individually included.  try the following 

    <properties>
      <sel-ver>3.8.1</sel-ver>
      <maven.compiler.target>1.8</maven.compiler.target>
      <maven.compiler.source>1.8</maven.compiler.source>
     </properties>

    <dependency>
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-java</artifactId>
       <version>${sel-ver}</version>
      </dependency>
      <dependency>
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-server</artifactId>
       <version>${sel-ver}</version>
      </dependency>

**Comment by snowymike** [comment]
in case it helps anyone else that stumbles upon this.

for me, this was broken with IEDriverServer 3.5.1 and selenium 3.5.3.  upgrading to IEDriverServer 3.8.0 did not fix it.  if i additionally upgraded to selenium 3.8.1 (with IEDriverServer 3.8.0), that did fix it.

to build with selenium 3.8.1, i had to add an explicit dependency for gson that i hadn't needed with selenium 3.5.3.  haven't seen this called out in the changelog or elsewhere.  i now have explicit dependencies both for guava and gson.

`    <dependencyManagement>
        <dependencies>
            
            <dependency>
                <groupId>com.google.guava</groupId>
                <artifactId>guava</artifactId>
                <version>23.0</version>
            </dependency>
            
			<dependency>
			  <groupId>com.google.code.gson</groupId>
			  <artifactId>gson</artifactId>
			  <version>2.8.2</version>
			</dependency>            
        </dependencies>
    </dependencyManagement>
`

i can't upgrade to selenium 3.8.1 yet due to various code changes i'll have to make to support it (deprecated and deleted methods), but regarding the mouseMoveTo UnsupportedCommandException, it is fixed in selenium 3.8.1 and IEDriverServer 3.8.0
